cmd/kubeadm/app/componentconfigs/kubelet.go

```diff
@@ -56,7 +56,11 @@ var kubeletHandler = handler{
 	GroupVersion: kubeletconfig.SchemeGroupVersion,
 	AddToScheme:  kubeletconfig.AddToScheme,
 	CreateEmpty: func() kubeadmapi.ComponentConfig {
-		return &kubeletConfig{}
+		return &kubeletConfig{
+			configBase: configBase{
+				GroupVersion: kubeletconfig.SchemeGroupVersion,
+			},
```

Comment:
If I got it right when a new version is created, the version should be hard-coded here, right?
might worth to add a comment?

Reply:
Well, in general that's not the only change, that's going to happen. Besides that, the developer who updates it needs to verify that the correct types are used in the `kubeletConfig` struct and that everything is marshaled accordingly.
So, in general, the whole `kubelet*.go` needs to be checked to see what needs bumping and what not.